packages/wdio-reporter/src/stats/runnable.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Main class for a runnable class (e.g. test, suite or a hook)
+ * mainly used to capture its running duration
+ */
+
+export interface Runnable {
+    uid?: string
+    title: string
+}
+
+export interface RunnableError {
+    message: string
+    stack: string
+    type: string
+}
+
+export default abstract class RunnableStats {
+    type: string
+    start: number
+    end?: number
+    private _duration: number
+
+    constructor (type: string) {
+        this.type = type
+        this.start = Date.now()
+        this._duration = 0
+    }
+
+    complete () {
```

Comment:
not sure if it makes sense to be explicit about no return values
```suggestion
    complete (): void {
```

Reply:
Return types are inferred. But if you want explicit return types on methods, you should enforce it via the tsconfig. In some cases it can be useful.